README.md

```diff
@@ -49,10 +49,14 @@ SPDX-License-Identifier: CC0-1.0
 1. Go to the [Releases page](../../releases)
 2. Download the latest `chrome-scripts-vX.X.X.zip` file
 3. Extract the zip file to a folder
-4. Open Chrome and navigate to `chrome://extensions`
-5. Enable **Developer mode** (top right corner)
-6. Click **Load unpacked**
-7. Select the extracted folder
+4. **Enable User Scripts API:**
+   - Open Chrome and navigate to `chrome://flags/#userscripts-api`
+   - Set **"Experimental UserScripts API"** to **Enabled**
```

Comment:
I think you can just open the details page of the extension after installing and enable the user scripts for the extension. please check this again?